### PR TITLE
CUI-7412 Fix arrow key navigation when FileUpload is within ActionBar

### PR DESCRIPTION
**Pull request by majornista**
## Description
When a coral-fileupload is included within a coral-actionbar-item in coral-actionbar, when focus moves to the coral-fileupload, arrow key navigation within the coral-actionbar no longer works.

## Related Issue
[CUI-7412](https://jira.corp.adobe.com/browse/CUI-7412)
[CUI-7409](https://jira.corp.adobe.com/browse/CUI-7409)
[CQ-4293594](https://jira.corp.adobe.com/browse/CQ-4293594)

## Motivation and Context
Fixes navigation within ActionBar, and activation of FileUpload from "... More" menu.

## How Has This Been Tested?
Tested in Coral-Spectrum example for ActionBar

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by majornista** [comment]
@indra2gurjar and @icaraps

**Comment by indra2gurjar** [comment]
Working as expected . Deployed at my instance http://igurjar:4502/assets.html/content/dam

**Comment by majornista** [comment]
LGTM, I am able to verify on http://igurjar:4502/assets.html/content/dam

**Comment by icaraps** [comment]
Is it possible add a unit test for this ?

**Comment by icaraps** [comment]
btw @indra2gurjar @majornista there seems to be a new flaky test: 
```
AnchorButton
    Implementation Details
      testButton
        Instantiation
          should be possible to clone a button with icon and size attribute using markup
            ✖ "before each" hook in "should be possible to clone a button with icon and size attribute using markup"
              Chrome Headless 83.0.4103.116 (Linux x86_64)
            Error: done() called multiple times in hook <AnchorButton Implementation Details testButton Instantiation should be possible to clone a button with icon and size attribute using markup
```
Do you know why it appeared so suddenly ?

**Comment by indra2gurjar** [comment]
@majornista  Here is the Unit Test. https://github.com/majornista/coral-spectrum/pull/1
Please merge.

**Comment by indra2gurjar** [comment]
@icaraps  Modified existing Unit test for the fixed use case  [here](https://github.com/adobe/coral-spectrum/pull/114/files#diff-cee31e756bf2979b56907eb655c1a4e8R590) Also fixed some failing tests.
Could you please have a look.

**Comment by icaraps** [comment]
If you do `cd coral-component-actionbar && gulp test` does it run without throwing errors ?

**Comment by indra2gurjar** [comment]
@icaraps  >If you do cd coral-component-actionbar && gulp test does it run without throwing errors ?

Yes. Locally tests are passing.

File: coral-component-actionbar/src/tests/test.ActionBar.js [hunk 1]
```diff
@@ -652,19 +668,20 @@ describe('ActionBar', function() {
       bar.primary.items.add({}).appendChild(new Button());
      
       // Wait for resize listener
-      window.setTimeout(() => {
+      window.setTimeout(function() {
```
Comment (icaraps) [inline coral-component-actionbar/src/tests/test.ActionBar.js:671]: Why do you use `function` instead of arrow function ?

**Comment by indra2gurjar** [comment]
@icaraps  Can we merge this?

**Pull request merged**